packages/graphql/schema.graphql

```diff
@@ -84,6 +96,7 @@ type LocalProject {
   projectId: String
   projectRoot: String!
   resolvedConfig: ResolvedConfig
+  specs(filterBy: SpecType): [Spec!]!
```

Comment:
This should likely be structured as a `SpecConnection` - we don't have this concept yet in the cypress codebase, but there are some decent patterns established in the cloud schema. Let's sync on this next week